server.py

```diff
@@ -47,6 +47,7 @@ class UrlType(enum.Enum):
 )
 
 # return the result of process.wait()
+@metrics_handler.track_subprocess_exit_code
```

Comment:
instead of a functools wrapper can we do
```py
process = subprocess.Popen(
    # ...
)
exit_code = process.wait()
MetricsHandler.ffmpeg_exit_codes.labels(
  exit_status=exit_code,
).inc()
```